sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -94,4 +94,20 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>3.0.0-M4</version> <!-- {x-version-update;org.apache.maven.plugins:maven-surefire-plugin;external_dependency} -->
+        <configuration>
+          <rerunFailingTestsCount>3</rerunFailingTestsCount>
```

Comment:
Generally we don't want to be retrying failing tests as this could mask flakiness in our clients.

Reply:
Right, will remove.